code/modules/reagents/reagents/toxin.dm

```diff
@@ -506,3 +506,70 @@
 
 /datum/reagent/toxin/xeno_growthtoxin/overdose_crit_process(mob/living/L, metabolism)
 	L.Losebreath(2)
+
+
+/datum/reagent/toxin/xeno_hemodile
```

Comment:
no way to counter this as a medic/doctor

no adverse reactions with other chems either?

Reply:
It is counterable through dylovene and hypervene that purge toxins.